### PR TITLE
[CARBONDATA-1266] Fixed issue for non existing table

### PR DESCRIPTION
**Pull request by jatin9896**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/325/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2911/

File: integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java [hunk 1]
```diff
@@ -251,7 +251,7 @@ private CarbonTable loadTableMetadata(SchemaTableName schemaTableName) {
 
       return parseCarbonMetadata(table);
     }
-    return null;
+    throw new RuntimeException("Table Doesn't Exist");
```
Comment (jackylk) [inline integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java]: should use `TableNotFoundException`

Comment (jackylk) [inline integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java]: And remove the null check in caller

**Comment by jatin9896** [comment]
@jackylk I did the required changes. Please check.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2926/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/340/

File: integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataMetadata.java [hunk 2]
```diff
@@ -115,9 +115,6 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName schemaTableName)
     }
 
     CarbonTable carbonTable = carbonTableReader.getTable(schemaTableName);
```
Comment (jackylk) [inline integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataMetadata.java]: I found there is one more place, how about line 150?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2933/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/347/

File: integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataMetadata.java (hunk 2 above)
Comment (jatin9896) [inline integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataMetadata.java]: Please check.

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**